src/pages/ResearcherProfile.js

```diff
@@ -375,28 +356,28 @@ export const ResearcherProfile = hh(class ResearcherProfile extends Component {
     return obj;
   };
 
-  saveProperties(profile) {
+  saveProperties = async (profile) => {
     Researcher.createProperties(profile).then(()=> {
```

Comment:
If you're modifying this to be an async/await function then you can take this a step further and get rid of the ```then``` blocks.

```
await Researcher.createProperties(profile)
await this.saveUser()
this.setState({...
```